build_ext/lint.py

```diff
@@ -359,6 +360,110 @@ def err(self, node, msg=None):
         return ret
 
 
+def detect_overindent(logical_line, tokens, indent_level, hang_closing, indent_char, noqa, verbose):
```

Comment:
Not sure how to do that.  I can cherry-pick it out, but I don't really want to just drop the commit via interactive rebase from this branch.  This feature branch is too massive to rebase off master (it's just conflict after conflict) so I've been merging master in.  What does git do if you merge in a branch that contains a commit that the feature branch already has?